decentraland/App.js

```diff
@@ -11,6 +11,7 @@ import {
   setMyAssetsList,
   setAccountCreationStatus,
   setAccountCreationActions,
+  addUserAction,
```

Comment:
We can keep this as-is, but I first read it as "the action of adding a user"

Reply:
(And I know it's my fault for calling it `userAction` haha)

Reply:
(Not gating merging this PR)

Reply:
Extracted to: https://github.com/tasitlabs/tasit/issues/342

Reply:
Thanks! Resolving this.